lib/beaker/hypervisor/docker.rb

```diff
@@ -552,13 +550,15 @@ def dockerfile_for(host)
       # Also, disable reverse DNS lookups to prevent every. single. ssh
       # operation taking 30 seconds while the lookup times out.
       # Also unbreak users with a bunch of SSH keys loaded in their keyring.
+      # Also unbreak CentOS9 & Fedora containers on Ubuntu 24.04 host (UsePAM no)
       dockerfile += <<~DF
-        RUN sed -ri \
+        RUN find /etc/ssh/sshd_config /etc/ssh/sshd_config.d/ -type f -exec sed -ri \
         -e 's/^#?PermitRootLogin .*/PermitRootLogin yes/' \
         -e 's/^#?PasswordAuthentication .*/PasswordAuthentication yes/' \
         -e 's/^#?UseDNS .*/UseDNS no/' \
+        -e 's/^#?UsePAM .*/UsePAM no/' \
         -e 's/^#?MaxAuthTries.*/MaxAuthTries 1000/' \
-        /etc/ssh/sshd_config
+        {} \\;
```

Comment:
there is a difference though, + would put every file in the command line
maybe for inline sed it doesn't matter, not sure

Reply:
You're right, but for `sed` is doesn't matter so it's actually more efficient when you don't have to start `sed` multiple times. In this case it really won't be noticeable, but there are cases where it can.